blockchain/consensus/deleg_cns/configs/delegator-config.toml

```diff
@@ -0,0 +1,51 @@
+[chain]
+name = "devnet"
+bootstrap_peers = []
+block_delay_secs = 30
+
+version_schedule = [
+  { height = "Skyr", version = "V16" }
+]
+
+height_infos = [
+  { height = "Breeze", epoch = -1 },
+  { height = "Smoke", epoch = -1 },
+  { height = "Ignition", epoch = -1 },
+  { height = "ActorsV2", epoch = -1 },
+  { height = "Tape", epoch = -1 },
+  { height = "Liftoff", epoch = -1 },
+  { height = "Kumquat", epoch = -1 },
+  { height = "Calico", epoch = -1 },
+  { height = "Persian", epoch = -1 },
+  { height = "Orange", epoch = -1 },
+  { height = "Claus", epoch = -1 },
+  { height = "Trust", epoch = -1 },
+  { height = "Norwegian", epoch = -1 },
+  { height = "Turbo", epoch = -1 },
+  { height = "Hyperdrive", epoch = -1 },
+  { height = "Chocolate", epoch = -1 },
+  { height = "OhSnap", epoch = -1 },
```

Comment:
Is it ok/safe to leave the pre `Skyr` heights at -1?

Reply:
I found the part that looks for the current network version [traverses](https://github.com/ChainSafe/forest/blob/59f9731e5f2706ce628b662488b5cee12ff653a6/types/networks/src/lib.rs#L245-L251) `height_infos` in reverse until it finds the first suitable entry. At least for that code it doesn't matter if the others entries would also be usable at height 0, as long as the entries are in correct order in the config file. 

I made sure everything has an entry that exists in the `calibnet` version, but I found it confusing that it has a [non-monotonic height](https://github.com/ChainSafe/forest/blob/59f9731e5f2706ce628b662488b5cee12ff653a6/types/networks/src/lib.rs#L245-L251) schedule, unlike the [mainnet variant](https://github.com/ChainSafe/forest/blob/f73daa61daa7779712d366928497f2a9d1d088eb/types/networks/src/mainnet/mod.rs#L35-L62). Maybe the order of the entries is not as significant as I thought and you can have some activated early, others later.